dev/managedservicesplatform/internal/stack/cloudrun/cloudrun.go

```diff
@@ -36,7 +36,9 @@ import (
 	"github.com/sourcegraph/sourcegraph/lib/pointers"
 )
 
-type CrossStackOutput struct{}
+type CrossStackOutput struct {
+	RedisID *string
```

Comment:
nit - lets make this a little more verbose for clarity

```suggestion
    // RedisInstanceID is the ID of the provisioned Redis instance,
    // and only set if one was requested.
	RedisInstanceID *string
```